### DISCUSSION TITLE
zh/guide/custom-tp-setting

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#706] [Announcements]
# zh/guide/custom-tp-setting

自定义传送介绍 可以自定义设置你想传送的地点,并且可以定时,快速切换,智能排序地点; 比如,你可以设定传送到每一个神瞳的地点然后自动拾取; 功能介绍 1 2 传送文件地址获取 社区分享; 传送文件地址 (https://discord.com/channels/1069057220802781265/1129339670447263755);

https://korepi.com/zh/guide/custom-tp-setting.html

**Comment by Micah123321** [comment]
测试评论

**Comment by DoublexQAQ** [comment]
6

**Reply by Micah123321** [reply]
牢底6啥
